### DISCUSSION TITLE
Publicity & Media 13 Feb 2024

### DISCUSSION THREAD

**Discussion by jimmy6DOF** [omigroup/media#70] [General]
TODO:
1. Develop a new preamble template for meeting records
2. RASI for media assets and tooling
3. Add  like minded bodies link to webpage (W3C,MSF,Goddot,M3,Angel,Etc++)
4. Decommission GitHub "webpage' repo
5. Poster - "New Discord"
6. Poster - "glTF Extensions"
7. Poster - "Next Event"
8. Fix Youtube info section

DONE:
1. Point Discord link from website to new server
2. Update OMI Logo for vote
3. Agree on meetup timeslot

**Comment by mrmetaverse** [comment]
As part of our 2/14/24 weekly, we are adding these items to the backlog. 

(check once the ticket is added to the backlog)

- [x] Develop a new preamble template for meeting records
- [ ] RASI for media assets and tooling
- [ ] Add like minded bodies link to webpage (W3C,MSF,Goddot,M3,Angel,Etc++)
- [ ] Decommission GitHub "webpage' repo
- [ ] Poster - "New Discord"
- [ ] Poster - "glTF Extensions"
- [ ] Poster - "Next Event"
- [ ] Fix Youtube info section

**Reply by mrmetaverse** [reply]
Develop a new preamble template for meeting records

**Reply by mrmetaverse** [reply]
RACI/RASI for media assets and tooling

**Reply by jimmy6DOF** [reply]
RASI Tracking Starter list :


|   |   |   |
|---|---|---|
|Resource|Section|go.to.who |
|Website|all|MTV|
|Discord|all|Aaron/MrM/|
| GitHub | all | MrM |
| Twitter | all | Jimmy6dof/Indiebio |
| LinkedIn | all | MrM |
| YouTube | all | MrM |
| Figma | all | MTV |
| Discord | Bot.01 | Aaron/MrM/ |

**Reply by mrmetaverse** [reply]
hey! can one of you add the remaining tickets to the backlog as we demoed last week? I can go over it again today if needed.
